app/views/divisions/index_with_member.html.haml

```diff
@@ -9,5 +9,5 @@
 
 %p All votes #{@member.full_name_no_electorate} could have attended.
 
-- @member.person.members.order(entered_house: :desc).each do |member|
-  = render "divisions", member: @member, divisions: member.divisions_possible.order(date: :desc, clock_time: :desc, name: :asc)
+= render "divisions", member: @member, divisions: @divisions
```

Comment:
A cleaner way to achieve this would have been to rebase this brance on top of master so you're working from the latest changes. Doesn't create a merge conflict so all good.